open-metadata-implementation/access-services/digital-service/digital-service-api/src/main/java/org/odpi/openmetadata/accessservices/digitalservice/ffdc/DigitalServiceErrorCode.java

```diff
@@ -0,0 +1,101 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.digitalservice.ffdc;
+
+import org.odpi.openmetadata.frameworks.auditlog.messagesets.ExceptionMessageDefinition;
+import org.odpi.openmetadata.frameworks.auditlog.messagesets.ExceptionMessageSet;
+
+
+/**
+ * The DigitalServiceErrorCode is used to define first failure data capture (FFDC) for errors that occur when working with
+ * the DigitalService OMAS Services.  It is used in conjunction with both Checked and Runtime (unchecked) exceptions.
+ *
+ * The 5 fields in the enum are:
+ * <ul>
+ *     <li>HTTP Error Code - for translating between REST and JAVA - Typically the numbers used are:</li>
+ *     <li><ul>
+ *         <li>500 - internal error</li>
+ *         <li>400 - invalid parameters</li>
+ *         <li>404 - not found</li>
+ *         <li>409 - data conflict errors - eg item already defined</li>
+ *     </ul></li>
+ *     <li>Error Message Id - to uniquely identify the message</li>
+ *     <li>Error Message Text - includes placeholder to allow additional values to be captured</li>
+ *     <li>SystemAction - describes the result of the error</li>
+ *     <li>UserAction - describes how a consumer should correct the error</li>
+ * </ul>
+ */
+public enum DigitalServiceErrorCode implements ExceptionMessageSet
+{
+    OMRS_NOT_INITIALIZED(404, "OMAS-DIGITAL-SERVICE-404-001",
+            "The open metadata repository services are not initialized for the {0} operation",
```

Comment:
when we refer to OMAS in the previous file we capitalize the text, for consistency maybe should 'Open Metadata Repository Services'